src/test/java/TestJewels.java

```diff
@@ -0,0 +1,30 @@
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+public class TestJewels {
+
+    private final ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+
+    @Before
+    public void setOut(){
+        System.setOut(new PrintStream(outputStream));
+    }
+
```

Comment:
you don't use ByteArrayOutputStream outputStream in test, why you declare it? Your test will be perfectly fine without @Beforre and @After